lib/hooks.dart

```diff
@@ -529,8 +529,10 @@ void useLayoutEffect(dynamic Function() sideEffect, [List<Object> dependencies])
 /// ```
 ///
 /// Learn more: <https://reactjs.org/docs/hooks-reference.html#useimperativehandle>.
-void useImperativeHandle(Ref ref, dynamic Function() createHandle, [List<dynamic> dependencies]) =>
-    React.useImperativeHandle(ref.jsRef, allowInterop(createHandle), dependencies);
+void useImperativeHandle(Ref ref, dynamic Function() createHandle, [List<dynamic> dependencies]) {
+  if (ref == null) return;
+  React.useImperativeHandle(ref.jsRef, allowInterop(createHandle), dependencies);
```

Comment:
Just out of curiousity, would there be a benefit to passing in `ref?.jsRef` and letting React's `useImperativeHandle` handle the `null` input? Or is it better to handle it on our end and no-op as soon as possible?

Reply:
I would vote to short-circuit before making an unnecessary interop call.